src/emulator/storage/rules/config.ts

```diff
@@ -49,18 +48,37 @@ export function getStorageRulesConfig(
 
     return getSourceFile(storageConfig.rules, options);
   }
-
   // Multiple targets
   const results: RulesConfig[] = [];
   const { rc } = options;
   for (const targetConfig of storageConfig) {
     if (!targetConfig.target) {
       throw new FirebaseError("Must supply 'target' in Storage configuration");
     }
-    rc.requireTarget(projectId, "storage", targetConfig.target);
-    rc.target(projectId, "storage", targetConfig.target).forEach((resource: string) => {
-      results.push({ resource, rules: getSourceFile(targetConfig.rules, options) });
-    });
+    const targets = rc.target(projectId, "storage", targetConfig.target);
+    if (targets.length === 0) {
+      // Fall back to open if this is a demo project
+      if (Constants.isDemoProject(projectId)) {
+        storageLogger.logLabeled(
+          "BULLET",
+          "storage",
+          `Detected demo project ID "${projectId}", using a default (open) rules configuration.`
```

Comment:
Super-mega-nit: Can we alter this log very slightly for debugging purposes so we know if it was logged here or in the above case?